DoLah/Commands/AddTaskCommand.cpp

```diff
@@ -7,25 +7,24 @@ namespace DoLah {
 
 	}
 
-	AddTaskCommand::~AddTaskCommand() {
-
+	AddTaskCommand::AddTaskCommand(DoLah::AbstractTask* task) {
+		task = task;
 	}
 
-	AddTaskCommand::AddTaskCommand(DoLah::Calendar cal, DoLah::AbstractTask* task) {
-		newTask = task;
-		calendar = cal;
+	AddTaskCommand::~AddTaskCommand() {
+
 	}
 
-	DoLah::AbstractTask* AddTaskCommand::getNewTask() {
-		return newTask;
+	DoLah::AbstractTask* AddTaskCommand::getTask() {
+		return task;
 	}
 
 	DoLah::Calendar AddTaskCommand::getCalendar() {
 		return calendar;
 	}
 
-	void AddTaskCommand::setNewTask(DoLah::AbstractTask* task) {
-		newTask = task;
+	void AddTaskCommand::setTask(DoLah::AbstractTask* task) {
```

Comment:
Command should not be allowed to set the task. For now once the task is injected into the command, Task object shouldn't be able to change.